go/api/base_client.go

```diff
@@ -7524,3 +7524,100 @@ func (client *baseClient) ZInterStoreWithOptions(
 	}
 	return handleIntResponse(result)
 }
+
+// / Blocks the connection until it pops and returns a member-score pair
+// with the highest score from the first non-empty sorted set.
+//
+// See [valkey.io] for details.
+//
+// Parameters:
+//
+//   - keys: An array of keys to check for elements.
```

Comment:
```suggestion
//   keys    - An array of keys to check for elements.
```
Similarly, please update for all parameters in this file. Please use other commands in this file as reference.